core/domain/exp_domain.py

```diff
@@ -808,6 +808,10 @@ def validate(self, strict=False):
         utils.require_valid_name(
             self.title, 'the exploration title', allow_empty=True)
 
+        if len(self.title) > 36:
```

Comment:
What's the plan for existing explorations with title > 36 characters? Wouldn't they throw an error now?

Reply:
Yes, you are right, I mistakenly raised up this PR before handling what to do with the beam job results. I think I'll close up this PR as of now and after handling the job results I'll reopen it.
Thanks for the review.